sympy/utilities/tests/test_timeutils.py

```diff
@@ -4,7 +4,7 @@
 
 def test_timed():
     result = timed(lambda: 1 + 1, limit=100000)
-    assert result[0] == 100000 and result[3] == "ns"
+    assert result[0] == 100000 and result[3] == "ns", str(result)
```

Comment:
Perhaps this change was unnecessary. The tests failed here on the previous commit but I couldn't reproduce the failure locally so I added this to get some debugging information. Afterwards the tests passed. I'll leave it there in case the test fails again.